configure.py

```diff
@@ -1554,7 +1554,7 @@ def MatchingFor(*versions):
     ActorRel(Matching,    "d_a_lbridge", extra_cflags=['-pragma "nosyminline on"']),
     ActorRel(Matching,    "d_a_leaflift", extra_cflags=['-pragma "nosyminline on"']),
     ActorRel(NonMatching, "d_a_lstair"),
-    ActorRel(NonMatching, "d_a_machine"),
+    ActorRel(NonMatching, "d_a_machine", extra_cflags=['-pragma "nosyminline on"']),
```

Comment:
```suggestion
    ActorRel(Matching,    "d_a_machine", extra_cflags=['-sym off']),
```

Using `-sym off` is what allows it to match on my side after I pull in your changes. Give that a shot.

Reply:
Thank you! Didn't work last time for some reason but showing 100% now.